src/cuda/gpu_get2e_subs.h

```diff
@@ -663,19 +762,124 @@ __device__ __forceinline__ void iclass_spdf10
      
      See M.Head-Gordon and J.A.Pople, Jchem.Phys., 89, No.9 (1988) for VRR algrithem details.
      */
-    //QUICKDouble store[STOREDIM*STOREDIM];
-    
+/*
+#if defined int_sp
+    QUICKDouble store[STOREDIM*STOREDIM];   
+#endif
+*/
     /*
      Initial the neccessary element for
      */
     
+#if defined int_sp 
```

Comment:
Why is this code replicated? It is identical for int_sp, int_spd, int_spdf, etc.

Reply:
I am aware of this. It was passed down from Yipu. Not sure why. I will have this cleaned up in a separate PR. There are some other useless stuff as well.